docs/rst/user_manual/configuration.rst

```diff
@@ -322,9 +322,9 @@ A complete example of all the configurations described on this page can be found
     # Discovery Server DDS Participant with ROS GuidPrefix so a local ROS 2 Client could connect to it
     # This Discovery Server will listen in ports 11600 and 11601 in localhost
 
-    simple:                             # Participant Id = simple
+    ServerROS2:                         # Participant Id = ServerROS2
```

Comment:
NIT:
This was called `simple` to exemplified that if `type` was specified, the type would not come from the Participant ID

Reply:
I find that to be a bit confusing, unless I add a comment with the explanation. But I wouldn't mention it since I don't see how a user would benefit from this.